components/contribution-flow/index.js

```diff
@@ -624,7 +634,10 @@ class ContributionFlow extends React.Component {
             return true;
           } else {
             const isCompleted = Boolean(noPaymentRequired || stepPayment);
-            if (isCompleted && stepPayment?.key === NEW_CREDIT_CARD_KEY) {
+            if (isCaptchaEnabled() && !stepProfile.captcha && !LoggedInUser) {
+              this.showError('Captcha is required.');
```

Comment:
With the latest API from react-intl it's now pretty simple to i18n this

```suggestion
              this.showError(intl.formatMessage({ defaultMessage: 'Captcha is required.' }));
```